hugin/gdocs/google_docs.py

```diff
@@ -0,0 +1,307 @@
+
+import gdata.spreadsheet.service
+import gdata.docs.service
+import gdata
+from gdata import MediaSource
+from gdata import GDataEntry
+import gdata.docs
+from hugin.gdocs import _from_unicode, _to_unicode
```

Comment:
methods defined with the prefix `_` are meant to be private, or only used within the module, so if you're going to use them here, it would be logical to name them just `from_unicode` and to_unicode``